### PR TITLE
PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalJetEnergySpectrum - Addding pT differential delta-pT histogram

### PR DESCRIPTION
**Pull request by aschmier**
PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalJetEnergySpectrum - Addding pT differential delta-pT histogram

### CODE REVIEW

File: PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalJetEnergySpectrum.cxx [hunk 1]
```diff
@@ -360,6 +363,11 @@ bool AliAnalysisTaskEmcalJetEnergySpectrum::Run(){
       }
     }
 
+    if(fDoBkgSub && datajets->GetRhoParameter() && fDoDifferentialDpT){
+        Double_t randomConePt = GetDeltaPtRandomCone();
+        fHistos->FillTH2("hPtJetVsRhoVsDeltaPtRC", ptjet, randomConePt, datajets->GetRhoVal());
```
Comment (mfasDa) [inline PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalJetEnergySpectrum.cxx:368]: Must be `FillTH3`, as the histogram is defined as 3D histogram.

**Review by mfasDa — CHANGES_REQUESTED** [review]
I think the histogram type is mixed up, otherwise fine.

**Comment by aschmier** [comment]
I fixed the mix-up, but I may not have squash-merged correctly.

**Comment by mfasDa** [comment]
Indeed commits are not squashed. Please do so locally and push again as I cannot do it from the webinterface for AliPhysics (for O2-related projects it is possible to squash there)

**Comment by aschmier** [comment]
Okay, should be good now

**Comment by mfasDa** [comment]
+1

**Pull request merged**